lua/nvim-treesitter/configs.lua

```diff
@@ -101,15 +102,18 @@ end
 local function disable_mod_conf_autocmd(mod, lang)
   if not config.modules[mod] or not M.is_enabled(mod, lang) then return end
 
-  local ft = parsers.lang_to_ft(lang)
-  api.nvim_command(string.format("autocmd! FileType %s", ft))
+  local cmd = string.format("lua require'nvim-treesitter.%s'.attach()", mod)
+  -- TODO(kyazdani): detach the correct autocmd... doesn't work when using %s, cmd
+  for _, ft in pairs(parsers.lang_to_ft(lang)) do
+    api.nvim_command(string.format("autocmd! NvimTreesitter FileType %s", ft))
+  end
```

Comment:
Can we have some comments on why that change ?

Reply:
hum basically before it didn't work because the augroup wasn't specified. Now it does, but i cannot remove a particular autocmd in the NvimTreesitter group, which causes TSDIsableAll to disable all modules autocmds...